.taskcluster.yml

```diff
@@ -0,0 +1,108 @@
+      && zip -g /artifacts/function.zip *.py
+      && cd /work_dir/
+      && pip install tox
+      && tox -e py36
+  in:
+    - $if: 'tasks_for == "github-pull-request" && event["action"] in ["opened", "reopened", "edited", "synchronize"]'
+      then:
+        $let:
+          head_branch: ${event.pull_request.head.ref}
+          head_rev: ${event.pull_request.head.sha}
+          repository: ${event.pull_request.head.repo.clone_url}
+          pull_request_number: ${event.pull_request.number}
+          pull_request_url: ${event.pull_request.html_url}
+        in:
+          created: ${created_on}
+          deadline: ${deadline_in}
+          expires: ${expires_in}
+          provisionerId: ${provisioner_id}
+          workerType: ${worker_type}
+          scopes: []
+          payload:
+            maxRunTime: {$eval: max_run_time_in_seconds}
+            image: ${docker_image}
+            command:
+              - /bin/bash
+              - --login
+              - -cx
+              - >-
+                git clone --branch ${head_branch} ${repository} /work_dir
+                && cd /work_dir
+                && git config advice.detachedHead false
+                && git checkout ${head_rev}
+                ${partial_command}
+            artifacts:
+              ${uploaded_artifact_path}:
+                expires: ${expires_in}
+                path: ${local_artifact_path}
+                type: file
+          metadata:
+            name: 'maven-lambda - Build and test (Pull Request #${pull_request_number})'
+            description: 'Testing and building maven-lambda - triggered by [#${pull_request_number}](${pull_request_url})'
+            owner: ${task_owner}
+            source: ${repository}/raw/${head_rev}/.taskcluster.yml
+    - $if: 'tasks_for == "github-push"'
+      then:
+        $let:
+          head_branch: ${event.ref}
+          head_rev: ${event.after}
+          repository: ${event.repository.clone_url}
+          compare_url: ${event.compare}
+        in:
+          created: ${created_on}
+          deadline: ${deadline_in}
+          expires: ${expires_in}
+          provisionerId: ${provisioner_id}
+          workerType: ${worker_type}
+          scopes: []
+          payload:
+            maxRunTime: {$eval: max_run_time_in_seconds}
+            image: ${docker_image}
+            command:
+              - /bin/bash
+              - --login
+              - -cx
+              - >-
+                git clone --branch ${head_branch} ${repository} /work_dir
+                && cd /work_dir
+                && git config advice.detachedHead false
+                && git checkout ${head_rev}
+                ${partial_command}
+            artifacts:
+              ${uploaded_artifact_path}:
+                expires: ${expires_in}
+                path: ${local_artifact_path}
+                type: file
+          metadata:
+            name: 'maven-lambda - Build and test (Pull Request #${pull_request_number})'
```

Comment:
Nit: I assume this is going to be the `master` branch so we might want to amend this title description?

Reply:
Oops, good catch. Fixed!